curriculum/outlines/Linear DS.md

```diff
@@ -0,0 +1,39 @@
+# Linear DS
+---
+## Outline
+1. Static Array
+	 1. 1D Static Array
+	 2. 2D Static Array
+	 3. 3D Static Array
+2. Dynamically-Resizeable Array
+3. BitSet
```

Comment:
- Core idea behind BitSets
- Boolean arrays vs BitSets
- important functions (mention them)

I think the core idea will lead to mention bitmasks. So, better be shifted after bitmasks.